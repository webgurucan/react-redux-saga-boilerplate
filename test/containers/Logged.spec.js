import expect, { createSpy } from 'expect';
import React from 'react';
import { mount } from 'enzyme';

import Logged from 'containers/Logged';

const dispatch = createSpy();

function setup() {
  const props = {
    dispatch,
    location: {}
  };

  return mount(<Logged {...props} />);
}

describe('Logged', () => {
  const wrapper = setup(true);

  it('should be a Component', () => {
    expect(wrapper.instance()).toBeA(React.Component);
  });

  it('should render properly', () => {
    expect(wrapper.find('.app__logged').length).toBe(1);
  });
});
