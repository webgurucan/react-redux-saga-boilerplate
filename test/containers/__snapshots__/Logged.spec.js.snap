exports[`Logged should render properly 1`] = `"<div class=\"app__logged app__route\"><div class=\"app__container\"></div></div>"`;
